modules/javafx.controls/src/test/java/test/javafx/scene/control/TreeTableCellTest.java

```diff
@@ -633,6 +633,25 @@ private void test_rt36715_cellPropertiesMirrorTableColumnProperties(
         cell.setSkin(new TreeTableCellSkin(cell));
     }
 
+    @Test
+    public void testCellInUneditableRowIsNotEditable() {
+        tree.setEditable(true);
```

Comment:
I recommend adding tests for cell.startEdit() using all combinations of TreeTableView, TreeTableColumn and TreeTableRow editable states.

Reply:
done